packages/test-runner-intercept/tsconfig.json

```diff
@@ -0,0 +1,77 @@
+// Don't edit this file directly. It is generated by generate-ts-configs script
+
+{
+  "extends": "../../tsconfig.node-base.json",
+  "compilerOptions": {
+    "module": "commonjs",
+    "outDir": "./dist",
+    "rootDir": "./src",
+    "composite": true,
+    "allowJs": true
+  },
+  "references": [
+    {
+      "path": "../parse5-utils/tsconfig.json"
+    },
+    {
+      "path": "../browser-logs/tsconfig.json"
+    },
+    {
+      "path": "../dev-server-core/tsconfig.json"
+    },
+    {
+      "path": "../test-runner-core/tsconfig.json"
+    },
+    {
+      "path": "../test-runner-coverage-v8/tsconfig.json"
+    },
+    {
+      "path": "../test-runner-mocha/tsconfig.json"
+    },
+    {
+      "path": "../test-runner-chrome/tsconfig.json"
+    },
+    {
+      "path": "../config-loader/tsconfig.json"
+    },
+    {
+      "path": "../dev-server-rollup/tsconfig.json"
+    },
+    {
+      "path": "../test-runner-playwright/tsconfig.json"
+    },
+    {
+      "path": "../test-runner-webdriver/tsconfig.json"
+    },
+    {
+      "path": "../polyfills-loader/tsconfig.json"
+    },
+    {
+      "path": "../dev-server-esbuild/tsconfig.json"
+    },
+    {
+      "path": "../dev-server-legacy/tsconfig.json"
+    },
+    {
+      "path": "../dev-server/tsconfig.json"
+    },
+    {
+      "path": "../test-runner-commands/tsconfig.json"
+    },
+    {
+      "path": "../test-runner-saucelabs/tsconfig.json"
```

Comment:
Do we need all these references in this tsconfig? I dont think thats needed

Reply:
I think you're right. I'll see if I can clean them. :-)

Reply:
This file is set by the `generate-ts-configs.mjs` script, but it bases itself on the deps and devDeps of the package. As I switched to `@web/test-runner-core` the list is now a lot smaller.
(fyi it also takes in transitive dependencies)